flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -283,12 +283,6 @@
 	@Deprecated
 	public static final String TASK_MANAGER_MEMORY_SEGMENT_SIZE_KEY = "taskmanager.memory.segment-size";
 
-	/**
-	 * The implementation to use for spillable/spilled intermediate results, which have both
-	 * synchronous and asynchronous implementations: "sync" or "async".
-	 */
-	public static final String TASK_MANAGER_NETWORK_DEFAULT_IO_MODE = "taskmanager.network.defaultIOMode";
```

Comment:
This is part of the public API (since `ConfigConstants` is `@Public`) and cannot be removed.

Reply:
@zhijiangW 
it seems we have to annotate those config constants as `@Deprecated` and add a doc `@deprecated` comment that the option is not used anymore.

Reply:
I see, thanks for the suggestion.